extension/content/customPersonaEditor.js

```diff
@@ -243,14 +254,14 @@ var CustomPersonaEditor = {
     PersonaService.changeToPersona(this.customPersona);
     switch (PersonaService.appInfo.ID) {
       case PersonaService.THUNDERBIRD_ID:
-	Cc['@mozilla.org/appshell/window-mediator;1'].
+    Cc['@mozilla.org/appshell/window-mediator;1'].
         getService(Ci.nsIWindowMediator).
         getMostRecentWindow("mail:3pane").
         document.getElementById("tabmail").
         removeCurrentTab();
         break;
       case PersonaService.FIREFOX_ID:
- 	window.close();
+    window.close();
```

Comment:
Same here, Github Desktop shows they have the same indent but Github.com complains.